projects/mongo-core-test.yml

```diff
@@ -0,0 +1,26 @@
+---
+runner: command
+params:
+  dir: mongo-core-test
+  setup: |
+    python3 -m venv .
+    COMPOSE_FILE='mongo5.yml' TEST_DIRECTORY=$(readlink -f ../mongo/jstests) docker compose up -d
```

Comment:
Using `export COMPOSE_FILE='mongo5.yml'` in setup doesn't seem to keep env vars for each individual tests